typingdna_activelock/README.md

```diff
@@ -0,0 +1,54 @@
+
+
+# TypingDNA ActiveLock
+
+## Overview
+
+[TypingDNA ActiveLock][3] is a Continuous Endpoint Authentication app that helps prevent unauthorized access to your company computers. Once installed on a user's PC, it continuously verifies the user by the way they type. If an unauthorized typing pattern is detected, ActiveLock can instantly lock the computer and log the data to your desired logging platform (such as Datadog).
+
+To visualize your data in Datadog, a custom ActiveLock app needs to be configured and installed. This is the same install for all of your company computers.
+
+
+## Setup
+
+### Configuration
+
+To generate a Datadog API key:
+
+1. Navigate to [Organization settings > API keys][4] in your Datadog account.
+2. Click **+ New Key** to generate an API key.
+
+To get your custom install app:
+
+1. Complete [this custom install form][7] by submitting your newly generated API key and Datadog site region (such as US1 or EU), along with your company details.
+2. You will then receive a custom ActiveLock app that you can install on your company computers and more information over email. This install has an initial limit of 10 seats, and comes with a default 30-day trial period. You will need to purchase a commercial ActiveLock license from TypingDNA in order to use it for more than 30 days, and/or on more than 10 computers. Contact [TypingDNA][2] for details or the reseller/partner through which you found us.
+3. After installation, your ActiveLock logs should start to appear in [Log Explorer][5].
+
+Note that if you came through a reseller/partner you should follow their instructions to get your custom install app (and commercial license).
```

Comment:
Will customers of Datadog and TypingDNA be from resellers as well?

Reply:
TypingDNA ActiveLock can be sold by resellers in which case, the same integration will be installed, BUT the reseller will be able to manage the license, changing it from trial to pro, add seats and expiration. Usually the reseller will provide all details separately to the client.